### ISSUE TITLE
Error while converting value to string

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#1392]
**Bug Description**
When replaying latest develop branch (https://github.com/bitshares/bitshares-core/tree/fc7ed0cdb327cb1378fa5d37e236a53d9b30e212), there are lots of  `ERROR_WHILE_CONVERTING_VALUE_TO_STRING` entries showing in log
```
2018-10-18T19:26:24        th_a:?unnamed?        push_proposal ] removing failed operation from applied_ops: ["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]                 db_block.cpp:313
2018-10-18T19:26:24        th_a:?unnamed? clear_expired_propos ] Failed to apply proposed transaction on its expiration. Deleting it.
["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]
2018-10-18T19:26:28        th_a:?unnamed?  check_for_blackswan ] *call_itr: ["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]                  db_update.cpp:233
2018-10-18T19:26:38        th_a:?unnamed?             do_apply ] between 100% and 0x100%:  ["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]                   account_evaluator.cpp:185
2018-10-18T19:26:38        th_a:?unnamed?             do_apply ] between 100% and 0x100%:  ["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]                   account_evaluator.cpp:185
2018-10-18T19:26:38        th_a:?unnamed?             do_apply ] between 100% and 0x100%:  ["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]                   account_evaluator.cpp:185
2018-10-18T19:26:38        th_a:?unnamed?             do_apply ] between 100% and 0x100%:  ["ERROR_WHILE_CONVERTING_VALUE_TO_STRING"]                   account_evaluator.cpp:185

```

I guess it might affect other functionalities E.G. API.

**Impacts**
Describe which portion(s) of BitShares Core may be impacted by this bug. Please tick at least one box.
- [ ] API (the application programming interface)
- [ ] Build (the build process or something prior to compiled code)
- [ ] CLI (the command line wallet)
- [ ] Deployment (the deployment process after building such as Docker, Travis, etc.)
- [ ] DEX (the Decentralized EXchange, market engine, etc.)
- [ ] P2P (the peer-to-peer network for transaction/block propagation)
- [ ] Performance (system or user efficiency, etc.)
- [ ] Protocol (the blockchain logic, consensus, validation, etc.)
- [ ] Security (the security of system or user data, etc.)
- [ ] UX (the User Experience)
- [x] Other (please add below): logging

**Steps To Reproduce**
Replay the blockchain.

**Expected Behavior**
Correct logging messages.

**Host Environment**
Please provide details about the host environment. Much of this information can be found running: `witness_node --version`. 
 - Host OS:             Ubuntu 16.04 LTS
 - Host Physical RAM    244GB
 - BitShares Version:   fc7ed0cdb327cb1378fa5d37e236a53d9b30e212
 - OpenSSL Version:     1.0.2g
 - Boost Version:       1.58
 
## CORE TEAM TASK LIST
- [ ] Evaluate / Prioritize Bug Report
- [ ] Refine User Stories / Requirements
- [ ] Define Test Cases
- [ ] Design / Develop Solution
- [ ] Perform QA/Testing
- [ ] Update Documentation

**Comment by oxarbitrage** [comment]
can be related to https://github.com/bitshares/bitshares-fc/pull/80

**Comment by pmconrad** [comment]
Cannot reproduce with boost-1.60. Please provide config.

The only instances of the "between 100%..." messages are these:
```
50843ms th_a       account_evaluator.cpp:185     do_apply             ] between 100% and 0x100%:  {"fee":{"amount":19029492,"asset_id":"1.3.0"},"registrar":"1.2.90713","referrer":"1.2.10331","referrer_percent":112,"name":"ff-test-7","owner":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS5WeuMmTCUzqkwcEo2TewSGhqQsi21ogVQ2hhG6ssHTmcsS7J29",1]],"address_auths":[]},"active":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS7isEia3qWYTggM4ZfdQQYbSKRq1sEC2fG6Bzip8yscKJqx1vvx",1]],"address_auths":[]},"options":{"memo_key":"BTS7isEia3qWYTggM4ZfdQQYbSKRq1sEC2fG6Bzip8yscKJqx1vvx","voting_account":"1.2.5","num_witness":0,"num_committee":0,"votes":[],"extensions":[]},"extensions":{}}
250846ms th_a       account_evaluator.cpp:185     do_apply             ] between 100% and 0x100%:  {"fee":{"amount":19030272,"asset_id":"1.3.0"},"registrar":"1.2.90713","referrer":"1.2.97634","referrer_percent":112,"name":"maverick3740","owner":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS54wyV4jZvgZTRVMmVX8ozJRWHZ6QhkLT1ryf6ehbP42Es4MoQZ",1]],"address_auths":[]},"active":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS5r2bN3FRU39QpuneEgevUNFUX2jNiE8QsBY4W1doifwwynuDcz",1]],"address_auths":[]},"options":{"memo_key":"BTS5r2bN3FRU39QpuneEgevUNFUX2jNiE8QsBY4W1doifwwynuDcz","voting_account":"1.2.5","num_witness":0,"num_committee":0,"votes":[],"extensions":[]},"extensions":{}}
250846ms th_a       account_evaluator.cpp:185     do_apply             ] between 100% and 0x100%:  {"fee":{"amount":19029492,"asset_id":"1.3.0"},"registrar":"1.2.90713","referrer":"1.2.96298","referrer_percent":112,"name":"kazubo-1","owner":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS7wUcD6Wn2QP461CQTHjkX7ptLzE5o1tYwBpQWUu2rkxLGoc8UL",1]],"address_auths":[]},"active":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS7VcPi35oqaL7ZqniwtwSSZsb42A5Lrd3RegAieozcAEV9HkmyR",1]],"address_auths":[]},"options":{"memo_key":"BTS7VcPi35oqaL7ZqniwtwSSZsb42A5Lrd3RegAieozcAEV9HkmyR","voting_account":"1.2.5","num_witness":0,"num_committee":0,"votes":[],"extensions":[]},"extensions":{}}
250856ms th_a       account_evaluator.cpp:185     do_apply             ] between 100% and 0x100%:  {"fee":{"amount":19029492,"asset_id":"1.3.0"},"registrar":"1.2.90713","referrer":"1.2.10331","referrer_percent":112,"name":"ff-test-8","owner":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS5DeAeiMPZG3K1QZVArZXTzYij2LHFmaTW2y8KhPMts7uqHDRmM",1]],"address_auths":[]},"active":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS8HfXi39vnhDyLuWxgCF5BCqoCaj4J51TGXxTfhTqc1Y1Dx6Z4L",1]],"address_auths":[]},"options":{"memo_key":"BTS8HfXi39vnhDyLuWxgCF5BCqoCaj4J51TGXxTfhTqc1Y1Dx6Z4L","voting_account":"1.2.5","num_witness":0,"num_committee":0,"votes":[],"extensions":[]},"extensions":{}}
251136ms th_a       account_evaluator.cpp:185     do_apply             ] between 100% and 0x100%:  {"fee":{"amount":9530468,"asset_id":"1.3.0"},"registrar":"1.2.17266","referrer":"1.2.32084","referrer_percent":208,"name":"malcolm-j-m-r","owner":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS4y3Ut999J8iS4TY5WKZhEvCdfAePMTZycozehNBmCGbdnb6Ms3",1]],"address_auths":[]},"active":{"weight_threshold":1,"account_auths":[],"key_auths":[["BTS8atChHTjjNqWSGtpND9Dr5nBFm3RPHDsSBHd6Cgw81hC1Vm9X4",1]],"address_auths":[]},"options":{"memo_key":"BTS8atChHTjjNqWSGtpND9Dr5nBFm3RPHDsSBHd6Cgw81hC1Vm9X4","voting_account":"1.2.5","num_witness":0,"num_committee":0,"votes":[],"extensions":[]},"extensions":{}}
```
Also see failed proposal and black swans in the usual format.

I do see the ERROR_WHILE_CONVERTING_VALUE_TO_STRING quite often in the P2P logfile.

**Comment by pmconrad** [comment]
Possible cause: ``max_object_depth`` is not initialized in appender config. Also can't be configured in config.ini / logging.ini.
https://github.com/bitshares/bitshares-core/blob/test-2.0.181018/libraries/app/config_util.cpp#L175

**Comment by abitmore** [comment]
Peter was correct. The logging messages in console shows correctly, but shows `ERROR_WHILE_CONVERTING_VALUE_TO_STRING ` in default.log file. I was using default generated config files. https://github.com/bitshares/bitshares-fc/pull/83 should fix it.

**Comment by abitmore** [comment]
Fixed by #1394.
